core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -568,8 +568,8 @@
   // accumulo garbage collector properties
   GC_PREFIX("gc.", null, PropertyType.PREFIX,
       "Properties in this category affect the behavior of the accumulo garbage collector."),
-  GC_CANDIDATE_BATCH_SIZE("gc.candidate.batch.size", "4000000", PropertyType.COUNT,
-          "The batch size used for garbage collection. The default is effectively 8MB"),
+  GC_CANDIDATE_BATCH_SIZE("gc.candidate.batch.size", "8m", PropertyType.BYTES,
```

Comment:
Although our type engine is case-insensitive, it is better to use upper-case, so it looks like MegaBytes, and not milliBytes. :smiley_cat:

```suggestion
  GC_CANDIDATE_BATCH_SIZE("gc.candidate.batch.size", "8M", PropertyType.BYTES,
```

Reply:
Fair, I had it capitalized but I didn't like the way it looked and saw we had a `1g` somewhere.

Reply:
I'll take a look and make a separate PR to standardize everything on the case used by SI conventions, for all of our property descriptions and default values (uppercase for everything, except kilo, which is lowercase). It probably doesn't matter to anybody but me. :smiley_cat: